extensions/wrapper/clients/typescript-client/README.md

```diff
@@ -24,7 +24,7 @@ You can find our API Wrapper Project
 
 ## How to install
 
-Requires a NodeJS / NPM project.
+Requires Node.js 18.
```

Comment:
You need Node.js 18 for development, but this documentation will be re-used for the NPM Package on npmjs.org.

You truly can only use the npm package if you have Node.js 18?

Reply:
Yes, if you would execute this in a non Node.js 18+ environment the global fetch API would not exist.
After running `npm run build` on the TS client and including the local dist dir in my test CLI it fails with Node.js 16.